src/transaction/types/input.rs

```diff
@@ -254,8 +248,12 @@ impl Input {
         match self {
             Self::CoinSigned { owner, .. }
             | Self::CoinPredicate { owner, .. }
-            | Self::MessageSigned { owner, .. }
-            | Self::MessagePredicate { owner, .. } => Some(owner),
+            | Self::MessageSigned {
+                recipient: owner, ..
```

Comment:
If we are to move one to the other, shouldn't we rename the attributes as well? Otherwise, it might be a bit confusing to handle the concepts.

Is the purpose of this to not cause a breaking change?

Reply:
This seems confusing to me as well, I think we should return none here.